### PR TITLE
[prometheus] Remove Grafana v8

### PR DESCRIPTION
**Pull request by scaps1**
## Description
As a way of keeping products that are included in our platform up to date, in this PR, we are dropping support for Grafana-v8.

## Why do we need it, and what problem does it solve?
This pull request removes Grafana version 8 from our project, transitioning solely to version 10. The upgrade is necessary to leverage enhanced features, performance improvements, and new security patches available in Grafana v10. By using the latest version, we ensure compatibility with modern infrastructures and continued support. Additionally, maintaining a single version simplifies management and reduces potential for conflicts. This change is a strategic step towards future-proofing our monitoring capabilities.

## Why do we need it in the patch release (if we do)?

To keep products that are included in our platform up to date.

## What is the expected result?
New and only version of Grafana

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: prometheus
type: chore
summary: Deprecate Grafana v8
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @scaps1. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @scaps1. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @scaps1. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Taior. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @scaps1. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by alexey-gavrilov-flant** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.27 11634528534 11634528534-1-con-1-27 /sys/deckhouse-oss/install:pr10359 redos@89.169.148.210

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @alexey-gavrilov-flant. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by alexey-gavrilov-flant** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.27 11626809908 11626809908-1-con-1-27 /sys/deckhouse-oss/install:pr10359 redos@130.193.51.77

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @alexey-gavrilov-flant. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @scaps1. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @scaps1. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by scaps1** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.27 11718471182 11718471182-3-con-1-27 /sys/deckhouse-oss/install:pr10359 redos@89.169.133.128

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @scaps1. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @scaps1. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Review by name212 — APPROVED** [review]

**Review by vladimirGuryanov — APPROVED** [review]

File: ee/modules/040-node-manager/capi/huaweicloud 2 [hunk 1]
```diff
@@ -0,0 +1 @@
+/deckhouse/modules/030-cloud-provider-huaweicloud/capi
```
Comment (sprait) [inline ee/modules/040-node-manager/capi/huaweicloud 2:1]: some trash?

**Review by sprait — CHANGES_REQUESTED** [review]
one thing

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vladimirGuryanov. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vladimirGuryanov. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Review by lazovskiy — APPROVED** [review]

**Review by nabokihms — APPROVED** [review]

**Pull request merged**